src/game/items/CItem.cpp

```diff
@@ -189,6 +189,19 @@ void CItem::DeleteCleanup(bool fForce)
 		}
 	}
 
+	//check if it is multi and clear player's list.
+	if (IsTypeMulti())
+	{
+		CItemMulti* pMulti = static_cast<CItemMulti*>(this);
+		CChar* pOwner = pMulti->GetOwner().CharFind();
+		if (pOwner)
+		{
```

Comment:
fix does not work..

![image](https://user-images.githubusercontent.com/20745578/199555632-6739f2fd-7a99-4bb8-b07d-d6c9711e3c87.png)

the check i marked never happens.

![image](https://user-images.githubusercontent.com/20745578/199555933-7d496635-6dbb-4b97-83b5-0c4e39f0205e.png)

Reply:
> Check if the char is a valid player please? M_pplayer could be null

if null pOwner would be null as well I think.

Reply:
I tried to fix this at CItem::delete() but pOwner is always null or not valid at this point. No idea why this happens. It can pass at CObj:: but not at CItem:: 
Any clue??